libs/core/src/lib/my-component/time-formatter/TimeFormatter.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import dayjs from 'dayjs';
+import { Palette } from 'libs/constants/src/lib/Palette';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    flexDirection: 'row',
+    backgroundColor: '#FFFFFF',
+    paddingHorizontal: 20,
+    paddingVertical: 5,
+    marginTop: 50,
+  },
+  text: {
+    textAlign: 'center',
+    marginRight: 10,
+    color: Palette.grey,
+  },
+});
+
+export interface TimeFormatterProps {
+  dateTime: Date;
+}
+
+export const TimeFormatter: React.FC<TimeFormatterProps> = ({ dateTime }) => {
```

Comment:
ტიპის მითითება გამოგრჩა
```suggestion
export const TimeFormatter: React.FC<TimeFormatterProps> = ({ dateTime }: TimeFormatterProps) => {
```